src/components/map/ha-locations-editor.ts

```diff
@@ -64,6 +64,9 @@ export class HaLocationsEditor extends LitElement {
   @property({ attribute: "theme-mode", type: String })
   public themeMode: ThemeMode = "auto";
 
+  @property({ type: Boolean, attribute: "move-on-click" })
+  public moveOnClick = false;
```

Comment:
this naming could be confusing.
What about

- updateOnClick
- pinOnClick
- ?

Reply:
pinOnClick sounds good, I hesitated with this current name as well.